examples/datadog-express-apm/container-agent/Dockerfile

```diff
@@ -1,19 +1,22 @@
-FROM node:18-alpine
+FROM node:20-alpine
 
 RUN apk add --no-cache tini git
 
 ENV NODE_ENV=production
 
 WORKDIR /usr/src/app
 
-COPY package.json yarn.lock /usr/src/app/
+COPY package.json yarn.lock ./
```

Comment:
Any reason why we are changing the WORKDIR?

Reply:
Thanks for the comment.
This line doesn't change the `WORKDIR`, It just copies those files to the current working directory, which is already set to `/usr/src/app` by the `WORKDIR` value on the line 7.